third_party/terraform/resources/resource_iam_binding.go.erb

```diff
@@ -186,7 +262,42 @@ func resourceIamBindingDelete(newUpdaterFunc newResourceIamUpdaterFunc, enableBa
 func getResourceIamBinding(d *schema.ResourceData) *cloudresourcemanager.Binding {
 	members := d.Get("members").(*schema.Set).List()
 	return &cloudresourcemanager.Binding{
-		Members: convertStringArr(members),
-		Role:    d.Get("role").(string),
+		Members:   convertStringArr(members),
+		Role:      d.Get("role").(string),
+<% unless version == 'ga' -%>
+		Condition: expandIamCondition(d.Get("condition")),
+<% end -%>
+	}
+}
+
+<% unless version == 'ga' -%>
+func expandIamCondition(v interface{}) *cloudresourcemanager.Expr {
+	l := v.([]interface{})
+	if len(l) == 0 || l[0] == nil {
+		return nil
+	}
+	original := l[0].(map[string]interface{})
+	return &cloudresourcemanager.Expr{
+		Description:     original["description"].(string),
+		Expression:      original["expression"].(string),
+		Title:           original["title"].(string),
+		ForceSendFields: []string{"Description", "Expression", "Title"},
+	}
+}
+
+func flattenIamCondition(condition *cloudresourcemanager.Expr) []map[string]interface{} {
+	if condition == nil {
+		return nil
+	}
+	if condition.Expression == "" && condition.Title == "" && condition.Description == "" {
```

Comment:
If title or expression are empty is it still a valid condition?

Reply:
No, but if it comes back from the API that way I don't think it's worth erroring